packages/iris-grid/src/mousehandlers/IrisGridContextMenuHandler.tsx

```diff
@@ -426,6 +427,26 @@ class IrisGridContextMenuHandler extends GridMouseHandler {
           });
         }
         filterMenu.actions.push(...this.nullFilterActions(column));
+      } else if (value === '') {
+        // empty string gets a special menu
+        if (quickFilters.get(modelColumn)) {
+          filterMenu.actions.push({
+            title: 'And',
+
+            actions: this.emptyStringFilterActions(
+              column,
+              valueText,
+              value,
```

Comment:
Why do we bother passing in `valueText` and `value`? We already know what the value and the text is, it's the empty string, as specified in the function name. No need to pass it in.